src/modm/driver/display/ssd1306_impl.hpp

```diff
@@ -80,105 +119,61 @@ modm::Ssd1306<I2cMaster, Height>::writeDisplay()
 	RF_END_RETURN(this->wasTransactionSuccessful());
 }
 
-template < class I2cMaster, uint8_t Height >
-modm::ResumableResult<bool>
+template<class I2cMaster, uint8_t Height>
+inline modm::ResumableResult<bool>
```

Comment:
```suggestion
modm::ResumableResult<bool>
```

Reply:
inline is only necessary for non-templated classes with method implementations in a header. Otherwise the method implementation is generated in multiple compile units and leads to linker errors. Aka. one definition rule.

Reply:
Check 🙌  also found more on this online...